lighthouse-core/gather/driver.js

```diff
@@ -743,6 +743,20 @@ class Driver {
   }
 
   /**
+   * Returns the flattened list of all DOM nodes within the document.
```

Comment:
I meant in regards to the "nodes" part...it's only returning nodes with `nodeType` 1. Could also just be "Returns the flattened list of all DOM elements within the document."